scalableminds/webknossos
# Fix Refreshing of Expired Datastore Tokens

Thank you @philippotto for the debugging help :)

### Steps to test:
- in application.conf, set silhouette.tokenAuthenticator.dataStoreExpiry = 1 minute
- trace some, wait a minute, trace some more
- new token should be requested, saving should succeed

### Issues:
- fixes #3607 

------
